src/component/CouponHeader.js

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+import {Text, View, StyleSheet} from 'react-native';
+import PropTypes from 'prop-types';
+import * as Constants from '../common/constants';
+
+export default class CouponHeader extends Component {
```

Comment:
CouponHeader should receive only the quantity of coupons. Then it should define the text logic (zero, one, more than one).